content/blog/2021/12/2021-12-03-election-results.adoc

```diff
@@ -0,0 +1,99 @@
+We also thank all candidates and voters who participated this year.
+
+Election results:
+
+* link:https://github.com/MarkEWaite[Mark Waite] joins the re-elected link:https://github.com/oleg-nenashev[Oleg Nenashev] and
+  link:https://github.com/kohsuke[Kohsuke Kawaguchi], link:https://github.com/halkeye[Gavin Mogan] and link:https://github.com/ewelinawilkosz[Ewelina Wilkosz]
+  on the link:/project/governance/#governance-board[Jenkins Governance Board]
+* link:https://github.com/timja[Tim Jacomb] will continue in the role of link:/project/team-leads/#release[Release Officer] (uncontested)
+* link:https://github.com/alyssat[Alyssa Tong] will become the new link:/project/team-leads/#events[Events Officer] (uncontested)
+* link:https://github.com/dduportal[Damien Duportal] was elected as the link:/project/team-leads/#infrastructure[Infrastructure Officer] (uncontested)
+* link:https://github.com/wadeck[Wadeck Follonier] was elected as the link:/project/team-leads/#security[Security Officer] (uncontested)
+* link:https://github.com/MarkEWaite[Mark Waite] was re-elected  in the role of link:/project/team-leads/#documentation[Documentation Officer] for another term
+
+The board positions and officer roles are an essential part of Jenkins' community governance and well-being,
+and we are excited to see contributors taking these roles.
+
+The board continues to require that no single company may have a majority on the Jenkins governance board.
+Oleg Nenashev is employed by Dynatrace, while Mark is employed by CloudBees.
+
+== Governance Board election details
+
+This year we had link:https://community.jenkins.io/t/jenkins-election-candidates/776#board-members-2[three candidates] participating in Jenkins Governance Board elections:
+Mark Waite,  Oleg Nenashev (returning), and Ullrich Hafner (returning)
+All of them are awesome community leaders who actively contribute to the Jenkins project and represent its users.
+It would be an honor to have them on the Jenkins board.
+Regardless of the election results, we appreciate their participation and the time they invested in these elections.
+
+This year we were electing 2 governance board members.
+We were using the https://civs.cs.cornell.edu/[Condorcet Internet Voting Service] that allows voters to rank their choices rather than just picking their one favorite choice.
+You can find full voting results link:https://civs.cs.cornell.edu/cgi-bin/results.pl?id=E_c206f1625b47c65d[here]:
+
+1. **Mark Waite** (Condorcet winner: wins contests with all other choices)
+2. **Oleg Nenashev** loses to Mark Waite by 37–22
+3. Ullrich Hafner loses to Mark Waite by 42–14, loses to Oleg Nenashev by 37–15
+
+Congratulations to the returning Oleg, and the newly elected Mark, and thanks to Ullrich for his past contributions.
+All new board members are elected for a 2-year term unless they decide to step down earlier.
+The estimated end of the term for them is December 02, 2022.
```

Comment:
Isn't it December 2023? due to the two terms for board positions?

Reply:
Yea I'm bad at numbers.

Feel free to update or I will eventually. It's been a busy daym

Reply:
Submitted as #4745